src/patternfly/components/Chip/chip.scss

```diff
@@ -14,29 +14,25 @@
   --pf-c-chip--m-overflow--PaddingLeft: 0;
   --pf-c-chip--m-overflow--BorderColor: transparent;
   --pf-c-chip--m-overflow--BorderWidth: 0;
-  --pf-c-chip--m-overflow--hover--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-chip--m-overflow--hover--BorderWidth: var(--pf-global--BorderWidth--sm);
 
   // modified chip button
-  --pf-c-chip--m-overflow--button--BorderRadius: var(--pf-global--BorderRadius--sm);
-  --pf-c-chip--m-overflow--button--BorderWidth: 0;
-  --pf-c-chip--m-overflow--button--BorderColor: transparent;
-  --pf-c-chip--m-overflow--button--PaddingRight: var(--pf-global--spacer--sm);
-  --pf-c-chip--m-overflow--button--PaddingLeft: var(--pf-global--spacer--sm);
-  --pf-c-chip--m-overflow--button--active--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--m-overflow--button--active--BorderColor: var(--pf-global--secondary-color--100);
-  --pf-c-chip--m-overflow--button--focus--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-chip--m-overflow--button--focus--BorderColor: var(--pf-global--secondary-color--100);
+  --pf-c-chip--m-overflow--c-button--BorderRadius: var(--pf-global--BorderRadius--sm);
+  --pf-c-chip--m-overflow--c-button--BorderWidth: 0;
+  --pf-c-chip--m-overflow--c-button--BorderColor: transparent;
```

Comment:
I think we can take this out. We are setting border to zero and it gets defined later on hover/active/focus.